README.md

```diff
@@ -131,19 +131,27 @@ disclosure related matters. If you need to contact us for another reason please
 
 See [BREAKING](./BREAKING.md).
 
+## Why Open Source?
+
+You might wonder why Authelia is open source while it adds a great deal of security and user experience to your infrastructure at zero cost. It is open source because we firmly believe that security should be available for all to benefit in the face the battlefield which is the Internet
```

Comment:
```suggestion
You might wonder why Authelia is open source while it adds a great deal of security and user experience to your infrastructure at zero cost.
It is open source because we firmly believe that security should be available for all to benefit in the face the battlefield which is the Internet
```